go.mod

```diff
@@ -3,13 +3,15 @@ module github.com/UniversityRadioYork/2016-site
 go 1.16
 
 require (
-	github.com/BurntSushi/toml v0.4.1
+	github.com/BurntSushi/toml v1.2.1
```

Comment:
this appears to break the website according to github

Reply:
I meant the github action logs

Reply:
fixed it, actions was running on an old version of Go that didn't support newer syntax (though I do need to check what prod uses)

Reply:
prod is 1.21, should be fine